tests/baseTestAPI.py

```diff
@@ -117,7 +120,7 @@ def setUpClass(cls):
 
         # Set the Agency for the agency user
         agencyUser = userDb.getUserByEmail(test_users['agency_user'])
-        agencyUser.agency = "testAgency"
+        agencyUser.cgac_code = "testAgency"
```

Comment:
I would agree it's more useful although not necessary since it's stored as a string in the database. For the unit tests, we need to use some code (something like we discussed as the default one we'd use internally) that is realistic but doesn't rely on something that's actually in the database.